appcues/src/main/java/com/appcues/debugger/screencapture/AndroidTargetingStrategy.kt

```diff
@@ -193,7 +209,7 @@ private fun SemanticsNode.asCaptureView(context: Context): ViewElement {
             height = unclippedGlobalBounds.height().toDp(density),
             displayName = it?.displayName,
             selector = it,
-            type = "Composable #$id",
+            type = it?.type ?: "Composable #$id",
```

Comment:
just a note, since type can be null here, i kept the "Composable #$id" which only appears on views that fail to provide a selector thus ignored by builder later on